tests/lib/rules/prop-types.js

```diff
@@ -1854,14 +1853,13 @@ ruleTester.run('prop-types', rule, {
       }
 
       A.propTypes = {
-        a: React.PropTypes.string,
-        ...SharedPropTypes // eslint-disable-line object-shorthand
-      };
-    `,
+         a: React.PropTypes.string,
```

Comment:
you've made this have 3 spaces of indentation from `A.propTypes =`; not that it really matters in tests, but can you revert the extra spaces?